Makefile

```diff
@@ -107,11 +107,11 @@ task:
 
 .PHONY: lint
 lint: golangci-lint ## Run golangci-lint linter & yamllint
-	$(GOLANGCI_LINT) run
+	$(GOLANGCI_LINT) run --enable gochecknoglobals
 
 .PHONY: lint-fix
 lint-fix: golangci-lint ## Run golangci-lint linter and perform fixes
-	$(GOLANGCI_LINT) run --fix
+	$(GOLANGCI_LINT) run --fix --enable gochecknoglobals
```

Comment:
I would suggest you to use `.golangci-lint.yaml` to configuring linters and linter settings. Check https://github.com/gardener/gardener-extension-registry-cache/blob/main/.golangci.yaml as example. You could remove the non-relevant configuration for the docforge.